napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -538,7 +246,7 @@ def show_warning(self, message: str = ""):
 
 
 class QPluginList(QListWidget):
```

Comment:
Not necessary for this PR: There is a minor UX (👀 @kandarpksk ) issue where the scroll list for the `Available Plugins` is reset to the top when the user clicks `Install`. This makes it hard to queue up multiple installs because the user keeps losing their place.

Reply:
Thanks! I hadn't noticed, but I agree we should create an issue to fix it in a different PR.

Reply:
Makes sense to me too. Tagging @isabela-pf who's working on the design of the napari Plugin Manager (the new name for this dialog/window)!